clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonClusterProvider.groovy

```diff
@@ -148,6 +145,9 @@ class AmazonClusterProvider implements ClusterProvider<AmazonCluster> {
       loadBalancers = translateLoadBalancers(allLoadBalancers)
       targetGroups = translateTargetGroups(allTargetGroups)
       serverGroups = translateServerGroups(allServerGroups)
+    } else {
+      Collection<CacheData> allServerGroups = resolveRelationshipDataForCollection(clusterData, SERVER_GROUPS.ns, RelationshipCacheFilter.include(LAUNCH_CONFIGS.ns))
```

Comment:
```
def sortedServerGroups = getServerGroups(application, account, clusterName, cloudProvider, null /* region */, false).findAll {
      def scopeMatch = it.region == scope || it.zones.contains(scope)
```

If `LAUNCH_CONFIGS` were not included, `it.zones` was null and the previously working check would NPE.

The big perf overhead comes with loading prematurely instances so I'd expect including `LAUNCH_CONFIGS` to not be material.